src/roadrunner-symfony-nyholm/composer.json

```diff
@@ -12,14 +12,17 @@
     "require": {
         "php": ">=8.0.5",
         "nyholm/psr7": "^1.4",
-        "spiral/roadrunner": "^2.0",
```

Comment:
What about kept this dependency and add the 2023 and 2024 versions in it? 

`
"spiral/roadrunner": "^2 || ^2023 || ^2024",
`

Reply:
See discussions before that is just incorrect and need handled differently.

Reply:
Thanks for the clarification!